components/lib/inputtext/InputText.js

```diff
@@ -70,6 +70,14 @@ export const InputText = React.memo(
         const isFilled = React.useMemo(() => ObjectUtils.isNotEmpty(props.value) || ObjectUtils.isNotEmpty(props.defaultValue), [props.value, props.defaultValue]);
         const hasTooltip = ObjectUtils.isNotEmpty(props.tooltip);
 
+        React.useEffect(() => {
+            if (isFilled) {
+                DomHandler.addClass(elementRef.current, 'p-filled');
+            } else {
+                DomHandler.removeClass(elementRef.current, 'p-filled');
+            }
+        }, [props.disabled, isFilled]);
```

Comment:
You use props.disabled here to trigger the hook but I don't see how that is used in the hook?

Reply:
props.disabled was included in useEffect dependencies because each time the component is enabled/disabled it needs to run that logic